consensus/src/commons.rs

```diff
@@ -139,35 +141,29 @@ impl QuorumMsgSender {
     }
 
     /// Sends an quorum (internally) to the quorum loop.
-    pub(crate) async fn send(&self, msg: Message) -> bool {
-        if let Payload::Quorum(q) = &msg.payload {
-            if q.validation.is_empty()
-                || q.ratification.is_empty()
-                || q.vote == Vote::NoCandidate
+    pub(crate) async fn send(&self, msg: Message) {
+        match &msg.payload {
             // TODO: Change me accordingly to https://github.com/dusk-network/rusk/issues/1268
+            Payload::Quorum(q)
+                if !q.validation.is_empty()
+                    && !q.ratification.is_empty()
+                    && q.vote != Vote::NoCandidate =>
```

Comment:
Do we need to propagate Quorum msg for either `Invalid` or `NoQuorum` Vote types?

Reply:
Yes, that's why there is a TODO above. I didn't changed that logic with this PR